src/applications/vaos/covid-19-vaccine/components/ConfirmationPageV2.jsx

```diff
@@ -33,16 +39,24 @@ function ConfirmationPageV2({
     return <Redirect to="/" />;
   }
 
-  const appointmentDateString =
-    moment(data.date1, 'YYYY-MM-DDTHH:mm:ssZ').format(
-      'dddd, MMMM D, YYYY [at] h:mm a ',
-    ) + getTimezoneAbbrBySystemId(systemId);
+  const timezone = getTimezoneBySystemId(systemId);
+  const momentDate = timezone
+    ? moment(slot.start).tz(timezone.timezone, true)
```

Comment:
What was the reason for switching the date to `slot.start` instead of `data.date1`?

Reply:
I didn't know both where the same before I made the change.